src/apps/lwaftr/conf.lua

```diff
@@ -48,7 +48,8 @@ local lwaftr_conf_spec = {
       policy_icmpv6_outgoing=Parser.enum_parser(policies),
       v4_vlan_tag=Parser.parse_vlan_tag,
       v6_vlan_tag=Parser.parse_vlan_tag,
-      vlan_tagging=Parser.parse_boolean
+      vlan_tagging=Parser.parse_boolean,
+      filter=Parser.parse_string,
```

Comment:
I think we need to be able to specify filters per-interface.  e.g. ipv4_ingress_filter, ipv4_egress_filter, ipv6_ingress_filter, ipv6_egress_filter